runner/app/pipelines/base.py

```diff
@@ -5,8 +5,9 @@
 class Pipeline(ABC):
     @abstractmethod
     def __init__(self, model_id: str, model_dir: str):
+        self.model_id: str # type hint so we can use the field in routes
```

Comment:
what does this comment mean?

```
# type hint so we can use the field in routes
```

Reply:
This avoids a typing error on all the `route/` implementations when they used the `model_id` field on the abstract `Pipeline` class (currently tons of typing errors on this proj, I'll try to clear them up as I make changes). 

I'm changing this comment to this which might be clearer?
```
# declare the field here so the type hint is available when using this abstract class
```